Nouveau environnement créé pour la QA de cette PR. Voici toutes mes procédures :
- Création de mon environnement avec l'installation de toutes les dépendances : OK.
- **Installation des fixtures : KO. Dépendance manquante : PyYAML.**
- Affichage de la barre de debug : OK
- Création d'un compte : OK
- Connexion : OK
- Affichage de toutes les alertes : OK
- Affichage de toutes les notifications : OK
- Affichage de tous les MPs : OK
- Création d'un MP : OK
- Ajout d'un membre : OK
- Quitter la conversation : OK
- Prévisualisation d'un message : OK
- Soumission d'un message : OK
- Test des balises du markdown à la Prévisualisation et à la soumission : OK
- Coloration du code : OK
- Création d'un tuto : OK
- Consultation d'un tuto hors ligne : OK
- Ajout d'un auteur : OK
- Edition d'un tuto : OK
- Mise en beta : OK
- Ajout de contenu et vérification d'une version différent entre la bêta et version offline : OK (git ok)
- Tous le processus de validation : OK
- Consulter un tutoriel en ligne : OK
- Téléchargement des formats d'export : OK (attention, je n'ai pas testé PDF, pas pandoc d'installé)

Erreur de l'installation des fixtures :

```
(zdsenv-1354)Mac-Pro-de-Gerard-2:zds-site gerard$ python manage.py loaddata fixtures/*.yaml
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 55, in handle
    self.loaddata(fixture_labels)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 84, in loaddata
    self.load_label(fixture_label)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 132, in load_label
    using=self.using, ignorenonexistent=self.ignore)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/serializers/__init__.py", line 133, in deserialize
    return d(stream_or_string, **options)
  File "/Users/gerard/.virtualenvs/zdsenv-1354/lib/python2.7/site-packages/django/core/serializers/__init__.py", line 49, in __call__
    raise self.exception
ImportError: Problem installing fixture '/Users/gerard/Documents/workspace-django/zds-site/fixtures/categories.yaml': No module named yaml
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -22,6 +22,7 @@ https://github.com/zestedesavoir/Python-ZMarkdown/archive/2.4.1-zds.10.zip
 easy-thumbnails==2.1
 
 # Development tools
+PyYAML==3.11
 django-debug-toolbar==1.2.1
 flake8==2.2.3
 autopep8==1.0.3

